internal/config/api.go

```diff
@@ -9,11 +9,18 @@ import (
 	"github.com/signadot/cli/internal/auth"
 	"github.com/signadot/cli/internal/buildinfo"
 	"github.com/signadot/go-sdk/client"
+	sdkauth "github.com/signadot/go-sdk/client/auth"
 	"github.com/signadot/go-sdk/transport"
 	"github.com/spf13/viper"
 	"gopkg.in/yaml.v2"
 )
 
+var (
+	AuthExpiredError    = errors.New("Authentication expired. Please log in using 'signadot auth login'")
```

Comment:
According to Go standards and static check, error variables should start with `Err`, so these should be renamed `ErrAuthExpired` etc.